go.mod

```diff
@@ -5,5 +5,6 @@ go 1.14
 require (
 	github.com/golang/protobuf v1.3.5 // indirect
 	github.com/hashicorp/terraform v0.12.9
+	github.com/mitchellh/mapstructure v1.1.2
```

Comment:
Is this some rando github? or a legit one?

Reply:
by legit i mean more like a large scale company or golang.

Reply:
mitchellh is legit core stuff generally

Reply:
cool cool. Thanks will :)